strings/trainer_sliding_strings.string

```diff
@@ -465,12 +465,142 @@ It's been a long time since I've had a\nbattle this close!\p
+M-My Rattata! How could you?\p
+
+#org @sText_TitleDefense_Joey_LastSwitchIn
+We're the best! We'll prove it!\p
+
+#org @sText_TitleDefense_Joey_LowHP
+My Pok\emon do their best when\nthey're backed into a corner!\p
+
+/// Selene
+#org @sText_TitleDefense_Selene_FirstMonDown
 This really takes me back to our\nfirst battle. Do you remember it,\ntoo?\p
 
-#org @sText_ChampionRematch_LastSwitchIn
+#org @sText_TitleDefense_Selene_LastSwitchIn
 It's quite refreshing to see how\nstrong a bond you share with your\nPok\emon.\p
 
-#org @sText_ChampionRematch_LowHP
+#org @sText_TitleDefense_Selene_LowHP
 I may not be the champion any\nlonger, but that doesn't mean I'll\ngo down easily!\p
+
+/// Alistair
+#org @sText_TitleDefense_Alistair_FirstMonDown
+So this is the strength that your\nbond has cultivated[.]\p
+
+#org @sText_TitleDefense_Alistair_LastSwitchIn
+We've grown closer, too.\nYou will acknowlege our strength!\p
+
+#org @sText_TitleDefense_Alistair_LowHP
+This isn't the end for us!\p
+
+/// Hawthorne
+#org @sText_TitleDefense_Hawthorne_FirstMonDown
+My, my[.] I never expected you would\nbe this strong!\p
+
+#org @sText_TitleDefense_Hawthorne_LastSwitchIn
+Hohoho! I see now what Selene saw\nin you so long ago.\p
+
+#org @sText_TitleDefense_Hawthorne_LowHP
+I'm grateful to have seen how\nmuch you've grown, [PLAYER].\p
+
+/// Clancy
+#org @sText_TitleDefense_Clancy_FirstMonDown
+My Pok\emon and I understand each\nother better now. You'll see!\p
+
+#org @sText_TitleDefense_Clancy_LastSwitchIn
+We're getting pumped up now[.]\p
+
+#org @sText_TitleDefense_Clancy_LowHP
+Time to bring this battle back to our\nfavour!\p
+
+/// Ena
+#org @sText_TitleDefense_Ena_FirstMonDown
+You've certainly been training since\nwe last battled!\p
+
+#org @sText_TitleDefense_Ena_LastSwitchIn
+We've also been training! If you let\nyour guard down, we'll win!\p
+
+#org @sText_TitleDefense_Ena_LowHP
+Uh oh, this isn't looking good for us.\p
+
+/// Copycat
+#org @sText_TitleDefense_Copycat_FirstMonDown
+Oh, I love it when you use that\nmove!\p
+
+#org @sText_TitleDefense_Copycat_LastSwitchIn
+You're good, but it's my turn to be\nKulure's Champion!\p
+
+#org @sText_TitleDefense_Copycat_LowHP
+So this is what it's like to be\npushed to the brink[.]\p
+
+/// Foreigner
+#org @sText_TitleDefense_Foreigner_FirstMonDown
+A lucky hit[.] I'm one of the best\ntrainers back home. I'll prove it!\p
+
+#org @sText_TitleDefense_Foreigner_LastSwitchIn
+Kulure's Pok\emon trainers are a cut\nabove the rest, huh[.]\p
+
+#org @sText_TitleDefense_Foreigner_LowHP
+Ngh! I'm not about to lose.\nNot like this!\p
+
+/// Collin
+#org @sText_TitleDefense_Collin_FirstMonDown
+You saw through that strategy, huh?\p
+
+#org @sText_TitleDefense_Collin_LastSwitchIn
+I've saved the best for last!\p
+
+#org @sText_TitleDefense_Collin_LowHP
+Heh, looks like I might not be\nclaiming your title after all[.]\p
+
+/// Crystal
+#org @sText_TitleDefense_Crystal_FirstMonDown
+CRYSTAL WRITE THIS
```

Comment:
@CrystalAQuick  I will need strings for you here.  Please makes sure they fit into two lines from this site:

https://cfru-text.herokuapp.com/

Reply:
Changes pushed up